specification/trafficmanager/resource-manager/Microsoft.Network/stable/2018-04-01/examples/HeatMapCountry-GET.json

```diff
@@ -0,0 +1,38 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "azuresdkfornetautoresttrafficmanager1323",
+    "profileName": "azuresdkfornetautoresttrafficmanager3880",
+    "heatMapType": "country",
+    "api-version": "2018-04-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/{subscription-id}/resourceGroups/azuresdkfornetautoresttrafficmanager1323/providers/Microsoft.Network/trafficManagerProfiles/azuresdkfornetautoresttrafficmanager3880/heatMaps/countryTrafficFraction",
+        "name": "country",
+        "type": "Microsoft.Network/trafficManagerProfiles/heatMaps/countryTrafficFraction",
```

Comment:
name, id, type values are all messed up. Please set up a sync with armapireview to get this approved.